### PR TITLE
New content from 2018/04/19

### PR DESCRIPTION
**Pull request by ig-ealbert**
* Grid fixes
* Remote virtualization demo
* Grid sorting demo
* DV changes from #291, #292, and #293
* Changed position of notes
* Updated resizing topic and added links (I know there is some English in here)

### CODE REVIEW

File: jp/components/grid_column_resizing.md [hunk 1]
```diff
@@ -5,9 +5,23 @@ _keywords: Ignite UI for Angular, UI コントロール, Angular ウィジェッ
 _language: ja
 ---
 
-### グリッドの列サイズ変更
+## グリッドの列サイズ変更
 
-**列のサイズ変更**も列ごとのレベルで有効化されます。つまり、**igx-grid** にサイズ変更可能な列およびサイズ変更可能ではない列の両方を含むことが可能です。`igx-column` の `resizable` 入力によって制御されます。**igx-grid** は遅延列のサイズ変更をサポートします。つまり、列はすぐにサイズ変更されません。サイズ変更インジケーターがドラッグ操作で一時的に表示され、ドラッグ操作が終了したときに新しい列サイズが適用されます。特定の列のサイズ変更はグリッド全体の幅に影響するため、残りの列の現在の幅は変更されません。
+<p class="highlight">Column resizing feature allows the user to change the width of a certain column at runtime. The `igx-grid` control supports **deferred column resizing**. Clicking on the right header border displays a temporary resize indicator that can be dragged left or right, serving as a preview. The column does not change its width while dragging the resize indicator. The new size is applied only when the drag operation is over. Resizing a certain column affects the overall width of the grid, so the rest of the columns will preserve their current widths.</p>
```
Comment (mpavlinov) [inline jp/components/grid_column_resizing.md]: @ig-ealbert This section is not translated. Is this on purpose?

Comment (ig-ealbert) [reply jp/components/grid_column_resizing.md]: @mpavlinov Yes, I wrote that in my pull request description.
"Updated resizing topic and added links (I know there is some English in here)"
Now there are conflicts, so I will try to resolve them.

Comment (mpavlinov) [reply jp/components/grid_column_resizing.md]: @ig-ealbert: Apologies. My bad. Didn't read the description :( .

**Comment by ig-ealbert** [comment]
I figured it out, it was because @mpavlinov made a merge, but still required more merges.  I didn't have Martin's merge.  I don't know if I did all of this correctly because it resulted in many merges.  If everything looks ok, let's go with it.  I don't particularly want to redo all the work from yesterday but I will if I have to.

**Comment by ig-ealbert** [comment]
It would be helpful if this could be reviewed and merged before I create a new pull request with the new content from yesterday.  Otherwise I'll be doing a large merge again.  I can do that if necessary.

**Comment by ig-ealbert** [comment]
I am going to continue to push to this branch for all changes from today.  In that case, the best approach may be to attempt to review and merge it before merging other pull requests to avoid more conflicts.

**Comment by ig-ealbert** [comment]
@mpavlinov @kdinev Please note that because of merges, there will be EN content in these files.  I know that and I will work on it today, hopefully to add the translations on Monday.  This happens because English content is added in the Japanese files.  This is the strategy requested by the development team so it is expected.  I need a day to translate the content and then it needs to be reviewed by Yoshimi.  That means at earliest I can add the updates the following day, but because of time differences it usually means 2 days for Bulgaria between when it's added and when my PR is available.

Also note that since updates come in so fast (ex. there were 15 closed PRs since the end of my yesterday), my updates will always be somewhat out of date.  As we've been doing, whatever work is completed goes into a PR and then I will create a new one for work after that PR is accepted.  There is no need to keep this open instead of merging it unless there is a critical issue, like it will break the build.  This is because any newly closed PRs will require me to merge in new content in English, and the cycle will continue.

**Pull request merged**